app/assets/stylesheets/componentLibraryOverrides.css

```diff
@@ -1,42 +1,41 @@
 :root {
-  --bs-border-radius: 0;
-  --bs-border-radius-sm: 0;
-  --bs-secondary-rgb: var(--stanford-palo-alto-dark-rgb);
+  --bs-secondary-rgb: var(--stanford-digital-blue);
```

Comment:
--bs-secondary-rgb is meant to be  of the form `108, 117, 125`, it shouldn't be a hex.

Reply:
It looks like we do have `--stanford-digital-blue-rgb` for this case

Reply:
Ohhh I guess we had it wrong originally. Thanks will fix!